modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -101,8 +123,13 @@ class HashTSDFVolumeCPU : public HashTSDFVolume
     cv::Vec3i volumeToVoxelCoord(cv::Point3f point) const;
 
    public:
-    //! Hashtable of individual smaller volume units
-    VolumeUnitMap volumeUnits;
+       Vec6f frameParams;
+       Mat pixNorms;
+       //VolumeUnitIndexes volumeUnitIndexes;
+       VolumeUnitIndexes volumeUnits;
+       cv::Mat volumes;
+       volumeIndex HashS;
```

Comment:
Please give more self descriptive names to `hashS` and `volumes`

Reply:
done